### ISSUE TITLE
Journeymap Bug

### ISSUE THREAD

**Issue by jrddunbr** [age-series/ElectricalAge#307]
```
[16:51:45] [JM-task-5/ERROR] [journeymap]: Couldn't get subblocks for block mods.eln.node.transparent.TransparentNodeBlock@7aae04fa: java.lang.NullPointerException: Parameter specified as non-null is null: method mods.eln.node.transparent.TransparentNodeBlock.func_149666_a, parameter tab
```

**Comment by jrddunbr** [comment]
L23 is the issue.

**Comment by jrddunbr** [comment]
Fixed in a6ab2ef
